Homework8/Task2/ExceptionGenerator.cs

```
@@ -0,0 +1,17 @@
+﻿namespace Homework8.Task2
+{
+    public class ExceptionGenerator
+    {
+        public void GenerateException()
```

In the task said - method should throw an exception but not to handle

---

Fixed